main.go

```diff
@@ -53,215 +60,346 @@ func NewClient(login, password string, httpClient *http.Client) *Client {
 	if httpClient == nil {
 		httpClient = &http.Client{
 			Transport: &http.Transport{
-				TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+				MaxIdleConnsPerHost: 20,
+				TLSClientConfig:     &tls.Config{InsecureSkipVerify: true},
 			},
-			Jar: cookies,
+			Jar:     cookies,
+			Timeout: 1 * time.Minute,
```

Comment:
Is there any motivation behind this particular timeout? Why not default one?

I mean I understand that it's like "a minute is enough for everyone" but it's more a code review-ish thing to ask since it's a question you might have when you look into a code next year and start thinking about what could be the reason of such a timeout "was it solving some specific issue? if so, then which one?". I suggest adding a comment here why you picked a minute or just use the default one.